### ISSUE TITLE
Improve decorations

### ISSUE THREAD

**Issue by burkeholland** [burkeholland/llm-api-tutor-example#4]
Figure out how long decorations should be and in what color to account for different themes

**Comment by isidorn** [comment]
Agreed. Current green colour does not give enough contrast in dark theme.

Maybe just copy what Debug is using for inline decorations

https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/debug/browser/debugEditorContribution.ts#L62

**Comment by isidorn** [comment]
Also maybe use some emoji on the start of the suggestion. Maybe 🤖  or 💡

**Comment by isidorn** [comment]
Definetley longer than 25 characters. I would try with 50.
